rfc8446.xml

```diff
@@ -1092,7 +1031,7 @@ target="supported-versions"/>) and the legacy_version field MUST be set to
 target="backward-compatibility"/> for details about backward
 compatibility.)</t>
   <t hangText='random'>
-  This field is 32 bytes generated by a secure random number generator.
+  32 bytes generated by a secure random number generator.
```

Comment:
We probably also want to change "This is a list of the symmetric cipher options" to just "A list of the symmetric cipher options" where it appears 15 lines below.  (Due to github limitations I can't comment on that line directly.)